src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -27,13 +30,25 @@ export const ConversationSpace: React.FunctionComponent = () => {
   const venue = useSelector(currentVenueSelectorData);
   const { recentVenueUsers } = useRecentVenueUsers({ venueName: venue?.name });
 
+  const { isShown: isChecked, toggle: toggleTables } = useShowHide();
```

Comment:
`isChecked` is not a self-explanatory name for such a complex component. -> `areNotAvailableTablesShown` ?
`toggleTables` -> maybe `toggleNotAvailableTables` ?

Reply:
fixed in e8298bec7e35a4e40e7175b130015db2080f52e5